app/main.py

```diff
@@ -1,3 +1,59 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int):
+        self.km = km
+
+    def __str__(self):
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other):
+        if isinstance(other, Distance):
+            return Distance(km=self.km + other.km)
+        raise TypeError("Operand must be of type 'Distance'")
```

Comment:
Consider returning `NotImplemented` instead of raising a `TypeError` when the operand is not a `Distance` instance. This allows Python to attempt the operation with the other operand's implementation.